sentinel/src/lib/feed.js

```diff
@@ -28,6 +44,15 @@ const registerFeed = seq => {
     .on('change', change => {
       if (!change.id.match(IDS_TO_IGNORE) &&
           !tombstoneUtils.isTombstoneId(change.id)) {
+
+        const queueSize = changeQueue.length();
+        if (queueSize >= MAX_QUEUE_SIZE) {
+          logger.info(`transitions: queue size ${queueSize} greater than ${MAX_QUEUE_SIZE}, we stop listening`);
```

Comment:
This doesn't need to be `info` I don't think. This is not information that is useful to read in the log, unless you're debugging, so that level is more appropriate.